rm_vk/rm_vk/Common/LikeControl/CustomLikeControl.swift

```diff
@@ -0,0 +1,110 @@
+// CustomLikeControl.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+@IBDesignable final class CustomLikeControl: UIControl {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let heartImageName = "suit.heart"
+        static let fillHeartImageName = "suit.heart.fill"
+        static let secondaryLabelColorName = "secondaryLabel"
+    }
+
+    // MARK: - Private Visual Components
+
+    private var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.distribution = .fillEqually
+        stackView.alignment = .center
+        return stackView
+    }()
+
+    private lazy var likeCountLabel: UILabel = {
+        let label = UILabel()
+        label.text = "\(likeCount)"
+        label.textAlignment = .center
+        label.textColor = UIColor(named: Constants.secondaryLabelColorName)
+        return label
+    }()
+
+    private lazy var likeButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(systemName: Constants.heartImageName), for: .normal)
+        button.tintColor = UIColor(named: Constants.secondaryLabelColorName)
+        button.addTarget(self, action: #selector(likeButtonAction), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Private Properties
+
+    private var isLike = false
+    private var likeCount = 0
+
+    @IBInspectable private var isLiked: Bool = false {
+        didSet {
+            updateLikeStatus()
+        }
+    }
+
+    // MARK: - Initializers
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setupView()
+    }
+
+    // MARK: - Lifecycle
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        stackView.frame = bounds
```

Comment:
```suggestion
        stackView.frame = bounds
```
может это нужно вынести в отдельный метод как во вьюдидлоад?